plugins/provisioners/ansible/provisioner/host.rb

```diff
@@ -72,33 +71,57 @@ def prepare_environment_variables
           @environment_variables["ANSIBLE_SSH_ARGS"] = ansible_ssh_args unless ansible_ssh_args.empty?
         end
 
+        def execute_command_from_host(command)
+          begin
+            result = Vagrant::Util::Subprocess.execute(*command) do |type, data|
+              if type == :stdout || type == :stderr
+                @machine.env.ui.detail(data, new_line: false, prefix: false)
+              end
+            end
+            raise Ansible::Errors::AnsibleCommandFailed if result.exit_code != 0
+          rescue Vagrant::Errors::CommandUnavailable
+            raise Ansible::Errors::AnsibleNotFoundOnHost
+          end
+        end
+
+        def execute_ansible_galaxy_from_host
+          command_values = {
+              :ROLE_FILE => get_galaxy_role_file(machine.env.root_path),
+              :ROLES_PATH => get_galaxy_roles_path(machine.env.root_path)
+            }
+          arg_separator = '__VAGRANT_ARG_SEPARATOR__'
```

Comment:
I don't think we usually use underscores either, just `VAGRANT_ARG_SEP` should be fine